### PR TITLE
[FAB-18281] Add an example for NewApplicationChannelGenesisBlock

### PR DESCRIPTION
**Pull request by satota2**
#### Type of change

- Test update
- Documentation update

#### Description

This patch adds an example for NewApplicationChannelGenesisBlock.
This patch also fixes minor typos.

#### Related issues

- https://jira.hyperledger.org/browse/FAB-18281

### CODE REVIEW

File: configtx/example_test.go [hunk 1]
```diff
@@ -561,6 +561,160 @@ func ExampleNewSystemChannelGenesisBlock() {
...
+				configtx.AdminsPolicyKey: {
+					Type: configtx.ImplicitMetaPolicyType,
+					Rule: "MAJORITY Admins",
+				},
+				configtx.BlockValidationPolicyKey: {
+					Type: configtx.ImplicitMetaPolicyType,
+					Rule: "ANY Writers",
+				},
+			},
+			OrdererType: orderer.ConsensusTypeSolo,
```
Comment (caod123) [inline configtx/example_test.go:585]: As this API will be used almost explicitly for the channel participation API/no system channel case which only supports etcdraft configurations, could you maybe update this example to use `ConsensusTypeEtcdRaft`?

**Review by caod123 — CHANGES_REQUESTED** [review]

**Comment by satota2** [comment]
@caod123 
Thank you for your reviews.

I updated the patch so that the example uses `ConsensusTypeEtcdRaft`.

**Review by caod123 — APPROVED** [review]

**Comment by caod123** [comment]
@satota2 looks like github actions is stuck checking the state of the PR, can you force push a no edit amendment to try and retrigger it? `git commit --amend --no-edit`

**Comment by satota2** [comment]
@caod123 
I tried to force push the PR. Is this OK?

**Review by caod123 — APPROVED** [review]

**Comment by caod123** [comment]
@btl5037 any ideas why this PR is stuck waiting for status report?

**Pull request merged**